### PR TITLE
scripts for updating collisions in moveit_config

### PR DESCRIPTION
**Pull request by fmessmer**
these helper simplify updating a moveit_config
to be used for https://github.com/ipa320/cob_robots/pull/581

### CODE REVIEW

File: cob_moveit_bringup/bin/add_robot.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```
Comment (mathias-luedtke) [inline cob_moveit_bringup/bin/add_robot.sh:1]: can you call the one from `cob_moveit_config` instead?

Comment (fmessmer) [reply cob_moveit_bringup/bin/add_robot.sh:1]: cob_moveit_config/add_robot.sh and cob_moveit_bringup/add_robot.sh are the same and both work...

**Comment by fmessmer** [comment]
plus fix warning for deprecated parameter namespace

File: cob_moveit_bringup/bin/add_robot.sh (hunk 1 above)
Comment (mathias-luedtke) [reply cob_moveit_bringup/bin/add_robot.sh:1]: > are the same and both work...

And I don't like copies.
```shell
#!/bin/bash
rosrun cob_moveit_config add_robot.sh "$@"
```

Comment (fmessmer) [reply cob_moveit_bringup/bin/add_robot.sh:1]: now I got what you meant...done

File: cob_moveit_bringup/bin/update_collisions.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+robot=$1
+
+moveit_config=`rospack find cob_moveit_config`
+config_pkg=$moveit_config/$robot
+
+echo "Updating collisions in "$config_pkg
+
+rosrun moveit_setup_assistant collisions_updater --config-pkg $config_pkg --default --always --keep --trials 10000
```
Comment (mathias-luedtke) [inline cob_moveit_bringup/bin/update_collisions.sh]: 10000 trials might be a little bit low.
One million?

Comment (fmessmer) [reply cob_moveit_bringup/bin/update_collisions.sh]: I chose 100000 as this is the maximum you can set on the slider in the GUI

**Review by mathias-luedtke — APPROVED** [review]

**Pull request merged**